optimism.tokenlist.json

```diff
@@ -637,6 +637,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x3C513dB8Bdc3806e4489d62C3d549A5Aaf6A4e97",
+      "name": "BitBTC",
+      "symbol": "BitBTC",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0xaBA2c5F108F7E820C049D5Af70B16ac266c8f128"
```

Comment:
What's the reason here for not using the standard bridge?

Reply:
To support L1 <=> L2 token mutual recognition, a custom bridge is required for BitBTC when new token was mined on L2 that should also be recognized by L1 according to [Optimism Document](https://community.optimism.io/docs/protocol/protocol-2.0.html#bridge-contracts). BitBTC(L2) was mined by l2Bridge when BitBTC(L1) is deposited to l1Bridge, and vice versa.

Reply:
Sorry it's taking longer as I need to review the custom bridge. Why is this using an upgradability pattern but at the same time the ownership of it has been renounced i.e. the owner is set to 0x0?

Reply:
1. The BitBTC protocol will be deployed on EVM-based chains. Currently, L1 <-> L2 cross-chain protocol is very mature, BitBTC will be recognized each other between L1 and L2 via cross-chain bridge.
2. The custom bridge of BitBTC needs a mining function, and the mining function can only be called by the gateway. We can only set one gateway, if we set the gateway to a specific one, then we can no longer support other L2(Arbitrum etc.), so we can only use the upgradeable proxy contract to implement the gateway in a multiple inheritance manner.
3. BitBTC’s owner, l2TokenBridge, and messenger have been correctly set up, please [check](https://etherscan.io/address/0xaBA2c5F108F7E820C049D5Af70B16ac266c8f128#readProxyContract)